pkg/config/config.go

```diff
@@ -733,6 +733,14 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("compliance_config.dir", "/etc/datadog-agent/compliance.d")
 	config.BindEnvAndSetDefault("compliance_config.cmd_port", 5010)
 
+	// Datadog security agent (runtime)
+	config.BindEnvAndSetDefault("runtime_security_config.enabled", false)
+	config.BindEnvAndSetDefault("runtime_security_config.debug", false)
```

Comment:
This is missing from the example config.

Reply:
Added documentation to configuration template